recipes/next-js-ssr/.dunner.yaml

```diff
@@ -0,0 +1,29 @@
+next:
+    - follow: init
```

Comment:
We don't need to run init steps always right? I feel it's better not follow init

Reply:
I agree, fixed :)